lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -114,7 +115,9 @@ sub _hmt_csv {
     my $hmt_ref = {};
 
     my $csv = Text::CSV->new({binary => 1}) or die "Cannot use CSV: " . Text::CSV->error_diag();
-    open $fh, '+>', undef or die "Could not open anonymous temp file - $!";                    ## no critic (RequireBriefOpen)
+    #open $fh, '+>', undef or die "Could not open anonymous temp file - $!";                       ## no critic (RequireBriefOpen)
```

Comment:
Please remove this if the code is no longer used

Reply:
remove old code, updated as suggested